app/Cargo.toml

```diff
@@ -76,9 +78,10 @@ glam = { workspace = true }
 rusty-hook = "^0.11.2"
 
 [features]
-default = []
+no_bundled_certs = []
+default = ["assimp"]
```

Comment:
Did this sneak in via merge?

Reply:
I conflicted with another commit, will go over this

Reply:
Yes, good catch, it was a result of the merge with Fredrik's commit to main and git auto merge.

Is fixed now. This was moved from `default` to `production` on main, but auto merge added both